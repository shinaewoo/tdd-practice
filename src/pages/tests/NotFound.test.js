import { render, screen } from "@testing-library/react";
import NotFound from "../NotFound";
import { withRouter } from "../../tests/utils";
import { Route } from "react-router-dom";

describe("NotFound", () => {
  it("renders correctly", () => {
    render(<NotFound />);

    expect(screen.getByText("Not Found!")).toBeInTheDocument();
  });
});

describe("NotFound2", () => {
  it("renders correctly2", () => {
    const { container } = render(
      withRouter(<Route path="/" element={<NotFound />} />)
    );

    expect(container).toMatchSnapshot();
  });
});
