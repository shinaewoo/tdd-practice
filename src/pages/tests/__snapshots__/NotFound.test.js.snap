// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound2 renders correctly2 1`] = `
<div>
  <div>
    Not Found!
  </div>
</div>
`;
